pytaxize/ids/ids.py

```diff
@@ -109,6 +110,16 @@ def itis(self, type="scientific"):
         self.db_ids = "itis"
         self.ids = dict(zip(self.name, out))
 
+    def gbif(self, rank="species"):
+        key = rank + "Key"
+        self.db_ids = "gbif"
+        self.ids = dict(
```

Comment:
was hoping we'd make the outputs consistent among the data sources (itis, ncbi, gbif, etc.) - that is, use `_make_id()` - requires taxon id, name, rank and the string for the data source, in this case `gbif`

Reply:
Not a problem- I didn't think the extra dependency would be worth it, but I suppose GBIF will also be used elsewhere in the repo, so I will use pygbif for the API call rather than making a request here.

Ah i see what that method is for now- for sure I can use the make_id function too. Will update PR now.

Reply:
thanks